src/utils/dnr-converter.js

```diff
@@ -9,131 +9,32 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0
  */
 
+import { convertWithAdguard } from '@ghostery/urlfilter2dnr';
 import { convert } from './dnr-converter-safari.js';
```

Comment:
Please move this into this file. This one became small enough, so we can include the whole `dnr-converter` in a single file.